test/unit/bin/ember-template-lint-test.js

```diff
@@ -0,0 +1,149 @@
+'use strict';
+
+var exec = require('child_process').exec;
+var assert = require('power-assert');
+var path = require('path');
+
+describe('ember-template-lint executable', function() {
+  describe('basic usage', function() {
+    describe('without any parameters', function() {
+      it('should exit without error and any console output', function(done) {
+        exec('./bin/ember-template-lint', function(err, stdout, stderr) {
+          assert.equal(err, null, 'exits without error');
+          assert.equal(stdout, '', 'doesn\'t print anything to standard output');
+          assert.equal(stderr, '', 'doesn\'t print anything to error output');
+          done();
+        });
+      });
+    });
+
+    describe('given path to non-existing file', function() {
+      it('should exit without error and any console output', function(done) {
+        exec('../../../bin/ember-template-lint app/templates/application-1.hbs', {
+          cwd: './test/fixtures/with-errors'
+        }, function(err, stdout, stderr) {
+          assert.equal(err, null, 'exits without error');
+          assert.equal(stdout, '', 'doesn\'t print anything to standard output');
+          assert.equal(stderr, '', 'doesn\'t print anything to error output');
+          done();
+        });
+      });
+    });
+
+    describe('given path to single file with errors', function() {
+      it('should print errors', function(done) {
+        exec('../../../bin/ember-template-lint app/templates/application.hbs', {
+          cwd: './test/fixtures/with-errors'
+        }, function(err, stdout, stderr) {
+          assert.ok(err, 'exits with error');
+          assert.ok(stdout, 'prints errors to standard output');
+          assert.equal(stderr, '', 'doesn\'t print anything to error output');
+          done();
+        });
+      });
+    });
+
+    describe('given wildcard path resolving to single file', function() {
+      it('should print errors', function(done) {
+        exec('../../../bin/ember-template-lint app/templates/*', {
+          cwd: './test/fixtures/with-errors'
+        }, function(err, stdout, stderr) {
+          assert.ok(err, 'exits with error');
+          assert.ok(stdout, 'print linting errors to standard output');
+          assert.equal(stderr, '', 'doesn\'t print anything to error output');
+          done();
+        });
+      });
+    });
+
+    describe('given directory path', function() {
+      it('should print errors', function(done) {
+        exec('../../../bin/ember-template-lint app', {
+          cwd: './test/fixtures/with-errors'
+        }, function(err, stdout, stderr) {
+          assert.ok(err, 'exits with error');
+          assert.ok(stdout, 'print linting errors to standard output');
```

Comment:
Or maybe just `prints errors to standard output` - I'll try to pass these descriptions through some grammar nazi before merging. 😆